### PR TITLE
 ⬆️ Upgrades UniFi Network Application to v6.5.54 (#318)

### PR DESCRIPTION
**Pull request by adamfeldman**
UniFi release info: https://community.ui.com/releases/UniFi-Network-Application-6-5-54/d717f241-48bb-4979-8b10-99db36ddabe1

# Proposed Changes

> (Describe the changes and rationale behind them)

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

**Comment by frenck** [comment]
image

This is a release candidate.

**Comment by AlecRust** [comment]
This release is [now stable](https://community.ui.com/releases/UniFi-Network-Application-6-5-54/d717f241-48bb-4979-8b10-99db36ddabe1#comment/6fa9e3f1-3b58-44cf-a55f-99ea287dd928).

File: unifi/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN \
         openjdk-8-jdk-headless=8u292-b10-0ubuntu1~20.04 \
     \
     && curl -J -L -o /tmp/unifi.deb \
-        "https://dl.ui.com/unifi/6.5.53/unifi_sysvinit_all.deb" \
+        "https://dl.ui.com/unifi/6.5.54-3b5d40203c/unifi_sysvinit_all.deb" \
```
Comment (frenck) [inline unifi/Dockerfile:21]: It has been promoted to stable:

```suggestion
        "https://dl.ui.com/unifi/6.5.54/unifi_sysvinit_all.deb" \
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by adamfeldman** [comment]
Pardon me for moving too quickly, I missed it was still in RC. Thank you @frenck!

**Review by frenck — APPROVED** [review]
Thanks, @adamfeldman 👍

**Pull request merged**